data/human/paradise world jobs.txt

```diff
@@ -521,3 +521,584 @@ mission "Paradise Job: Wilderness Retreat"
+	job
+	repeat
+	passengers 2 6
+	clearance "When you inform the spaceport controller that your passengers are permitted to live on <destination>, they give you the clearance to land."
+	to offer
+		has "event: initial deployment 1"
+		not "event: initial deployment 2"
+		random < 30
+	source Martini
+	destination
+		distance 4 12
+		not attributes "station"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog phrase "generic passenger dropoff payment"
+		payment 8000 80
+
+
+mission "Martini Bombings (Evacuation, Exhausted Search and Rescue Team)"
+	name "Transport discharged search and rescue team to <planet>"
+	description "These <bunks> members of a search and rescue team from <planet> have been searching for victims of the nuclear bombings in Martini's capital city, and now have been discharged. Transport them back to their home on <planet>, where they will pay you <payment>."
+	job
+	repeat
+	passengers 2 6
+	clearance "When you inform the spaceport controller that your passengers are permitted to live on <destination>, they give you the clearance to land."
+	to offer
+		has "event: initial deployment 1"
+		not "event: initial deployment 2"
+		random < 70
+	source Martini
+	destination
+		distance 4 12
+		not attributes "station"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog phrase "generic passenger dropoff payment"
+		payment 8000 80
+
+
+
+# Martini evac on complete dialog
+phrase "martini evac dialog singular"
+	word
+		"Throughout the whole trip, your passenger from Martini has been "
+	phrase
+		"martini evac dialog"
+
+phrase "martini evac dialog plural"
+	word
+		"Throughout the whole trip, your passengers from Martini have been "
+	phrase
+		"martini evac dialog"
+
+phrase "insults for southerners"
+	word
+		"'those "
+		"'them "
+	word
+		""
+		"bloody"
+		"damn"
+		"jumpy"
+		"upstart"
+		"whining"
+		"stupid"
+		"seditious"
+		"treasonous"
+		"terrorizing"
+		"greedy"
+	word
+		" "
+	word
+		"southerners"
+		"collectivists from Bourne"
+		"subversives from the Rim"
+		"insurrectionists from the Rim"
+		"charity-cases from the South"
+		"rocketeers from Solace"
+		"pirates from Greenrock"
+		"unlawfuls from Thule"
+		"gunrunners from Rust"
+	word
+		"'"
+
+phrase "martini evac dialog"
+	word
+		"accusing"
+		"repeatedly accusing"
+	word
+		" "
+	word
+		"the Free Worlds"
+		"the Pact"
+		"the Southern Pact"
+		"${insults for southerners}"
+	word
+		" for the "
+	word
+		"nuclear bombings"
+		"destruction"
+		"devastation"
+	word
+		" on Martini"
+		" of their home city"
+	word
+		". They hand you <payment> and head off for their"
+	word
+		" " 10
+		" well-kept "
+		" expensive "
+		" opulent "
+		" exorbitant "
+		" summer "
+		" vacation "
+		" extra "
+		" other "
+		" friend's "
+		" relative's "
+	word
+		"penthouse"
+		"condo"
+		"apartment"
+		"mansion"
+		"house"
+	word
+		" on <planet>."
+
+
+
+mission "Martini Bombings [Medical 1]"
```

Comment:
What's with the brackets? The previous missions used `()`, these ones use `[]`.

Reply:
Generally I use `[]` for numbers and `()` for text. I will change it to `"Martini Bombings Medical [#]`